src/sentry/runner/commands/devserver.py

```diff
@@ -6,6 +6,25 @@
 
 from sentry.runner.decorators import configuration, log_options
 
+_DEFAULT_DAEMONS = {
+    "worker": ["sentry", "run", "worker", "-c", "1", "--autoreload"],
+    "cron": ["sentry", "run", "cron", "--autoreload"],
+    "post-process-forwarder": [
+        "sentry",
+        "run",
+        "post-process-forwarder",
+        "--loglevel=debug",
+        "--commit-batch-size=1",
+    ],
```

Comment:
I was thinking maybe leave this one out of `_DEFAULT_DAEMONS` and inline it? It's kind of verbose and I don't have a scenario where it would be skipped.

Reply:
I think it's better to just have all the daemons in one place. It's less surprising that way.